pkg/app/master/command/tui/cli.go

```diff
@@ -18,7 +19,12 @@ var CLI = &cli.Command{
 	Aliases: []string{Alias},
 	Usage:   Usage,
 	Action: func(ctx *cli.Context) error {
-		m, _ := home.InitialModel()
+		gcvalues, ok := command.CLIContextGet(ctx.Context, command.GlobalParams).(*command.GenericParams)
+		if !ok || gcvalues == nil {
+			return command.ErrNoGlobalParams
+		}
+
+		m, _ := home.InitialModel(gcvalues)
```

Comment:
let's use a less Bubble Tea specific name... just call it `InitialTUI` (`home.InitialTUI`).

Reply:
Done. Resolved here - https://github.com/mintoolkit/mint/pull/55/commits/24ff10acd9fc80e73f91d19eed06a7e9f23e4c6e